sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1463,4 +1463,11 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
         |FROM (SELECT '{"f1": "value1", "f2": 12}' json, 'hello' as str) test
       """.stripMargin), Row("value1", "12", BigDecimal("3.14"), "hello"))
   }
+
+  test("SPARK-13056: Null in map value causes NPE") {
+    val df = Seq(1 -> Map("abc" -> "somestring", "cba" -> null)).toDF("key", "value")
+    df.registerTempTable("maptest")
```

Comment:
do we need to put the test in hive module? I think `DataFrameSuite` in sql core module is a good place to put this test, we can just test `df.select($"value"("abc"))` instead of registering a temp table.